moto/awslambda/urls.py

```diff
@@ -12,5 +12,7 @@
     '{0}/(?P<api_version>[^/]+)/functions/(?P<function_name>[\w_-]+)/?$': response.function,
     '{0}/(?P<api_version>[^/]+)/functions/(?P<function_name>[\w_-]+)/invocations/?$': response.invoke,
     '{0}/(?P<api_version>[^/]+)/functions/(?P<function_name>[\w_-]+)/invoke-async/?$': response.invoke_async,
-    '{0}/(?P<api_version>[^/]+)/tags/(?P<resource_arn>.+)': response.tag
+    '{0}/(?P<api_version>[^/]+)/tags/(?P<resource_arn>.+)': response.tag,
+    '{0}/(?P<api_version>[^/]+)/functions/(?P<function_name>[\w_-]+)/policy?$': response.policy,
```

Comment:
Is there a way I can merge these last two statements?  I've tried
`{0}/(?P<api_version>[^/]+)/functions/(?P<function_name>[\w_-]+)/policy(/?P<sid>[\w_-]+)?

but it doesn't capture the case when the sid is provided.